addons/web/static/tests/views/kanban_tests.js

```diff
@@ -5986,29 +5986,29 @@ QUnit.module('Views', {
     QUnit.test('progress bar subgroup count recompute', async function (assert) {
```

Comment:
It is just a rewrite of this test, selecting elements through vanilla JS…
And yet this enabled the test to pass.

Reply:
Maybe I should try to investigate more on this